src/DssDirectDepositHub.sol

```diff
@@ -336,6 +352,7 @@ contract DssDirectDepositHub {
                 fees = availableAssets;
             }
             ilk.pool.withdraw(fees);
+            daiJoin.join(address(ilk.pool), fees);
```

Comment:
Same here. I think the `join` can be done to `address(this)` and then `move` from it.